sklearn/tree/tests/test_tree.py

```diff
@@ -1640,3 +1640,60 @@ def _pickle_copy(obj):
             assert_equal(typename, typename_)
             assert_equal(n_outputs, n_outputs_)
             assert_equal(n_samples, n_samples_)
+
+
+def test_multilabel_invariance():
+
+    def list_list(lis):
+        return [list(arr) for arr in list(lis)]
+
+    for name, y_ml_list in product(CLF_TREES, (list(y_multilabel),
```

Comment:
Maybe I've inadvertently sent you on a wild goose chase, but if lists of lists and lists of arrays are supported as a multioutput format for trees/forests, that should be sufficient to indicate that they are supported as a multilabel format, especially if there are other tests that ensure the multilabel case works at all. If there are tests for different multioutput formats, they suffice.